### DISCUSSION TITLE
This PR was not automatically merged despite it's a patch having "Automerge: Enabled"?

### DISCUSSION THREAD

**Discussion by gremo** [renovatebot/renovate#9576] [Request Help]
This PR was not automatically merged: https://github.com/gremo/nest-winston/pull/336 despite automerge seems enabled:

![Cattura](https://user-images.githubusercontent.com/1532616/114986776-b55c3700-9e94-11eb-8728-45d364c4366a.PNG)

Don't know what I did wrong since `automergeMinor` and `automergePatch` presents are enabled:

```json
{
  "extends": [
    "config:base",
    ":automergeMinor",
    ":automergePatch",
    ":disablePeerDependencies",
    ":pinDevDependencies",
    ":semanticCommitsDisabled"
  ],
  "packageRules": [
    {
      "matchPaths": ["package.json"],
      "enabled": false
    },
    {
      "matchFiles": ["package.json"],
      "enabled": true
    },
    {
      "matchDepTypes": ["dependencies"],
      "rangeStrategy": "bump"
    }
  ]
}
```

**Comment by viceice** [comment]
Please check the logs there should be a reason.

Renovate needs two runs to successfully do an automerge.

**Reply by rarkins** [reply]
Looks like no tests?

**Reply by gremo** [reply]
Hi @viceice, I was checking the logs. Since I'm pretty new to Renovate I'm not sure how to read this. Here is what I've found:

> DEBUG: Checking if we can automerge branch(branch="renovate/ts-jest-26.5.x")
> DEBUG: mergeStatus=no automerge(branch="renovate/ts-jest-26.5.x")
> DEBUG: PR is configured for automerge(branch="renovate/ts-jest-26.5.x")

**Reply by viceice** [reply]
@gremo please provide more full log for that pr.

I think @rarkins is right.

**Reply by gremo** [reply]
Why you say "no tests" @rarkins @viceice? Are we talking about project tests? https://github.com/gremo/nest-winston/blob/master/package.json#L19

Here is the full log: https://pastebin.com/QsrgEHAV

**Reply by viceice** [reply]
You need automatic ci checks for automerge.

**WARNING** ⚠️

You can disable required status checks but this can break your main branch!

**Reply by gremo** [reply]
@viceice so Travis CI is not enough?  https://travis-ci.org/gremo/nest-winston and https://github.com/gremo/nest-winston/blob/master/.travis.yml

**Reply by viceice** [reply]
Your logs do missing status checks from travis. Is there a missing travis setting?

**Reply by gremo** [reply]
@viceice Honestly, I don't know how to say to Renovate "look, this is the Travis CI status for this branch". I was thinking that was a fully automatic process / configuration.

In addition, the branch of the PR was tested OK yesterday (https://travis-ci.org/github/gremo/nest-winston/builds/767190460):

![Cattura](https://user-images.githubusercontent.com/1532616/115008194-1ee74000-9eab-11eb-8fd2-dcb733828180.PNG)

To resume:

- [x] Tests are present
- [x] Travis CI present and active
- [x] Branch was tested OK
- [x] PR shows "Automerge: Enabled"
- [x] Renovate config seems OK

Still can't figure out why automerge didn't worked 😢

**Reply by rarkins** [reply]
Travis needs to report them back to GitHub as status checks or check runs. Otherwise it's unsafe to merge

**Reply by gremo** [reply]
@rarkins OK I see thanks, do you know where I can read about this? Any other repository using Renovate + Travis CI to look for?

**Reply by rarkins** [reply]
You have a Travis-GitHub problem and not a Travis-Renovate problem. See https://peterpoliwoda.me/travis-checks-in-github-continuous-integration/ for an example of how the check runs should look when Travis is correctly reporting results to GitHub PRs

**Reply by viceice** [reply]
Chech the travis docs, maybe you need the travis github app

https://docs.travis-ci.com/user/migrate/open-source-on-travis-ci-com/

**Reply by gremo** [reply]
I see that my old account is on Travis CI (org) and app was not installed for that repository. I'll try to setup the github Travis CI application (https://github.com/marketplace/travis-ci) on my repository and retry that PR, hope this works.

**Reply by gremo** [reply]
I migrated to travis-ci.com, installed the app and now the PR shows the integration correctly. I've then rebased it but still can't automerge? Or I need to do it manually now?

![pr](https://user-images.githubusercontent.com/1532616/115020027-42fe4d80-9eba-11eb-91ab-d4ce4479304d.PNG)


Job log here https://pastebin.com/bEugA24T

**Reply by rarkins** [reply]
You'll need to wait until the next time Renovate runs on the repo. It's not triggered immediately by the status checks but instead usually checks every 1-2 hours. If the PR is up to date with the main branch and tests pass, then it will be auto merged

**Reply by gremo** [reply]
I'll wait then, **thank you for your patience** @rarkins and @viceice!
